Automattic/wp-calypso
# Activate unified mailbox forms

This set of changes makes the unified mailbox form active, replacing the former mailbox forms.

Note: The domain upsell still retains the old comparison view, with the legacy forms, while we fix a pesky [navigation situation](pbLl1t-1a5-p2).

#### Proposed Changes

* Enable all the new unified form features behind the `unify-mailbox-forms` flag
* Remove subsequently unused legacy form components
* Pass the `newQuantity` extra field in the cart to fix a bug where it's passed for new subscriptions

#### Testing Instructions



Scenario 1 - ( performed for both Professional Email and Google Workspace  separately )
* Have a domain without an email subscription
* Navigate to [ **Upgrades** ] -> [ **Emails** ] -> [ **Add Email** ]
* Purchase one or multiple mailboxes
* Confirm that the provisioning was successful

Scenario 2 - ( performed for both Professional Email and Google Workspace separately )
* Have a domain without an existing email subscription
* Navigate to [ **Upgrades** ] -> [ **Emails** ] -> [ `<:domain>` ]
* Purchase additional mailboxes
* Confirm that the additional provisioning was successful

For both scenarios, the form validation should also work as expected.
